mobile/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -137,27 +137,27 @@
     <string name="settings_service_ip_monitor_monitor_root">Netlink 監視 (root)</string>
     <string name="settings_service_ip_monitor_poll">輪詢</string>
     <string name="settings_service_ip_monitor_poll_root">輪詢 (root)</string>
-    <string name="settings_service_upstream_monitor">目前上游接口</string>
+    <string name="settings_service_upstream_monitor">目前上游</string>
     <string name="settings_service_upstream_monitor_summary">%1$s；後援：%2$s</string>
     <string name="settings_service_upstream">上游網路介面</string>
     <string name="settings_service_upstream_auto">自動檢測系統 VPN 服務</string>
     <string name="settings_upstream_fallback">後援上游介面</string>
     <string name="settings_upstream_fallback_auto">自動檢測系統預設網路</string>
     <string name="settings_service_clean">清理/重新套用路由規則</string>
-    <string name="settings_service_clean_summary">將修改的設定套用到目前啟用的服務上。也可用於修復偶爾會發生的競態條件。</string>
+    <string name="settings_service_clean_summary">將修改的設定套用到目前啟用的服務上，也可用於修復偶爾會發生的競態條件</string>
     <string name="settings_service_dhcp_workaround">啟用 DHCP 因應措施</string>
-    <string name="settings_service_dhcp_workaround_summary">如果裝置無法取得 IP 位址，嘗試開啟這選項。</string>
+    <string name="settings_service_dhcp_workaround_summary">如果裝置無法取得 IP 位址，嘗試開啟這選項</string>
     <string name="settings_system_tether_offload">網路共用硬體加速</string>
     <string name="settings_system_tether_offload_summary">系統「開發人員選項」的捷徑</string>
     <string name="settings_misc">雜項</string>
     <string name="settings_help">說明</string>
     <string name="settings_misc_logcat">匯出偵錯資訊</string>
-    <string name="settings_misc_logcat_summary">這種非常有用啊 (^O^)／</string>
+    <string name="settings_misc_logcat_summary">如此有用 (^0^)</string>
     <string name="settings_misc_source">專案首頁 @ GitHub</string>
-    <string name="settings_misc_source_summary">閱讀使用說明，star，提交 issues，合作</string>
-    <string name="settings_misc_donate">抖內</string>
+    <string name="settings_misc_source_summary">閱讀手冊、加星號、提交問題並參與</string>
+    <string name="settings_misc_donate">捐助</string>
```

Comment:
This part is an additional supplement to be more friendly to users. No need to update translations.